packages/abi-coder/src/utils/constants.ts

```diff
@@ -1,18 +1,37 @@
+// ******************** //
+// Types and Type Regex //
+// ******************** //
```

Comment:
Although I'm a fan of evident code chunk annotations, I'd suggest using a more standard multiline comment with a non-variable opening/closing line length. In my experience, reaching a consensus on these things is often unrealistic as people's opinions differ significantly, hence why standards may be preferable.

```ts
/*
  Types and Type Regex
*/
```

Reply:
[212fe9a](https://github.com/FuelLabs/fuels-ts/pull/1780/commits/212fe9aef80fb898cc38de370f669c41939346b2)

Reply:
I think I got a bit trigger happy with the asterisk key by this point 😆

Reply:
Honestly, I used to have extravagant chunk dividers with ** going til the column limit _(in my case, usually 80 chars)_.

I had to force myself to make this "go standard" suggestion here.

You do what you gotta do, I guess. 😂